libs/util/Caches.arucas

```diff
@@ -0,0 +1,75 @@
+import Assert from util.Auxilary;
+import AsymmetricBiMap from util.Collections;
+/*
+*   Class Cache - basic Cache object that removes items sequentially
+*/
+class Cache : Map {
+    var map : Map;
+    var MAX_SIZE : Number = 16;
+    Cache() : super() {
+        this.map = {};
+    }
+    Cache(MAX_SIZE : Number) : super() {
+        Assert.assert(MAX_SIZE > 0);
+        this.map = {};
+        this.MAX_SIZE = MAX_SIZE;
+    }
+    fun put(key, value) {
+        if (!this.containsKey(key)) {
+            super.put(key, value);
+            this.checkMAX_SIZE();
+        }
+        else {
+            super.put(key, value);
+        }
+    }
+    fun checkMAX_SIZE() {

```

Comment:
FUNCTIONS SHOULD FOLLOW CAMEL CASING.

Reply:
notepad moment oof
I forgot to change it to capacity